src/lib/features/feature-search/feature-search-store.ts

```diff
@@ -351,6 +356,48 @@ class FeatureSearchStore implements IFeatureSearchStore {
         };
     }
 
+    private findFeatureStrategies(queryBuilder: Knex.QueryBuilder) {
```

Comment:
Does this need to actually return something or should it be renamed to something like `applyFindFeatureStrategies` to signal that the outcome of calling the method is "stored" in the QueryBuilder argument passed in?

Reply:
Yep, renamed